claimsponsor/claimsponsor_test.go

```diff
@@ -0,0 +1,223 @@
+package claimsponsor
+
+import (
+	"fmt"
+	"math/big"
+	"path"
+	"testing"
+	"time"
+
+	"github.com/agglayer/aggkit/log"
+	tree "github.com/agglayer/aggkit/tree/types"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/russross/meddler"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	TestRetryAfterErrorPeriod      = 2 * time.Second
+	TestMaxRetryAttemptsAfterError = 3
+	TestWaitTxToBeMinedPeriod      = 5 * time.Second
+	TestWaitOnEmptyQueue           = 10 * time.Second
```

Comment:
Nitpick:
in case these are used only in this file, I would make them private

```suggestion
	testRetryAfterErrorPeriod      = 2 * time.Second
	testMaxRetryAttemptsAfterError = 3
	testWaitTxToBeMinedPeriod      = 5 * time.Second
	testWaitOnEmptyQueue           = 10 * time.Second
```